### ISSUE TITLE
openGL-related error on linux

### ISSUE THREAD

**Issue by Eiim** [Arisotura/melonDS#518]
I've just finished setting up melonDS but when I try to run it I get this:

melonDS 0.8.3
http://melonds.kuribo64.net/

** (melonDS:17438): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-adcjbIE7AQ: Connection refused
melonds/melonDS: symbol lookup error: melonds/melonDS: undefined symbol: gdk_gl_context_set_use_es

It seems like the warning is probably harmless and unrelated to the failure at the bottom, which was the reason it failed to launch. My architecture is x86_64, and I'm running Ubuntu 16.04.

**Comment by Arisotura** [comment]
welp. I wonder what kind of circumstances can cause this error. might just be old OpenGL/etc? but I'd need to investigate it.

in the meantime, if you know how to compile things: you can comment out the gdk_gl_context_set_use_es() call, which is in src/libui_sdl/libui/unix/gl.c on line 75.

**Comment by PoroCYon** [comment]
Sounds like either old OpenGL drivers or an ancient GPU to me.

**Comment by v1993** [comment]
This should not happen on QT frontend anymore.

**Comment by WaluigiWare64** [comment]
We don't use libui anymore.
